packages/composites/linear-finance/src/config/index.ts

```diff
@@ -1,10 +1,11 @@
 import { Config as BaseConfig } from '@chainlink/types'
 import { Requester } from '@chainlink/ea-bootstrap'
-import xbci from './indices/xbci'
-import xlci from './indices/xlci'
+import xbci from './../indices/xbci'
+import xlci from './../indices/xlci'
```

Comment:
Could be cleaner to remove pathing to the cwd
```suggestion
import xbci from '../indices/xlci'
```